.eslintrc.cjs

```diff
@@ -1,14 +1,81 @@
 module.exports = {
-  env: { browser: true, es2020: true },
+  env: {
+    browser: true,
+    es2020: true,
+    jest: true,
+  },
   extends: [
     'eslint:recommended',
     'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
     'plugin:react-hooks/recommended',
+    'plugin:react/jsx-runtime',
+    'airbnb',
+    'airbnb-typescript/base',
+    'plugin:prettier/recommended',
   ],
   parser: '@typescript-eslint/parser',
-  parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
-  plugins: ['react-refresh'],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
+  plugins: [
+    '@typescript-eslint',
+    'react',
+    'react-refresh',
+    'unused-imports',
+    'prefer-arrow',
+    'prettier',
+  ],
   rules: {
+    'prettier/prettier': ['error'],
     'react-refresh/only-export-components': 'warn',
+    "@typescript-eslint/no-unused-vars": ["error"],
+    // "react/prop-types": "off",
```

Comment:
Remove commented code. It is bad practice to leave commented code when merging.

Reply:
Fixed